feelpp/pyfeelpp/tests/conftest.py

```diff
@@ -39,7 +39,7 @@ def __init__(self,config):
 
 @pytest.fixture(scope="session")
 def init_feelpp():
-    return InitFeelpp(feelpp.globalRepository("pyfeelpp-tests"))
+    return InitFeelpp(feelpp.globalRepository("pyfeelpp-tests")).e
```

Comment:
what is e?

Reply:
e is the Feelpp Environment (data of the class InitFeelpp)
then the tests have access to the environment interface